/*Crie uma situação matemática na qual os números precisam ser informados pelo usuário.
Além disso, deve ocorrer uma exceção dentro do código. Utilize os métodos try(), catch() e finally() 
para realizar a captura e o tratamento dessa exceção.*/

function name(params) {
    let 
    //
    if (condition) {
        
    } else {
        throw new Error()
    }
}

try {
    
} catch (error) {
    
}finally{

}